### ISSUE TITLE
Wrong icons without font-lock+

### ISSUE THREAD

**Issue by a13** [domtronn/all-the-icons.el#105]
Since font-lock+ was removed by 52d1f2d36468146c93aaf11399f581401a233306, all-the-icons shows icon using default font, so some icons are broken.
![all-the-fail](https://user-images.githubusercontent.com/251267/35919391-ff713e80-0c25-11e8-8d7f-2652de6c55d0.png)

Loading font-lock+ fixes the problem.

![all-the-win](https://user-images.githubusercontent.com/251267/35919514-68d9f646-0c26-11e8-9841-48a6ca03025a.png)

**Comment by a13** [comment]
evaluation of `font-lock-default-unfontify-region` from font-lock+ fixes the problem

**Comment by a13** [comment]
original `font-lock-default-unfontify-region` ignores `font-lock-ignore` (lol) property

**Comment by a13** [comment]
btw, instead of redefining `font-lock-default-unfontify-region` we can set `font-lock-unfontify-region-function`

**Comment by a13** [comment]
well, I decided to use quelpa as a package builder for font-lock+
```
(use-package font-lock+
  :ensure t
  :quelpa
  (font-lock+ :repo "emacsmirror/font-lock-plus" :fetcher github))
```

**Comment by a13** [comment]
there's a fix for the problem #106
